### ISSUE TITLE
[Problem - BUG] Abillities misscalculated

### ISSUE THREAD

**Issue by vasilegherasim** [IroncladGames/SinsOfADarkAge#959]
As shown in the image attached, there are slight differencies between the total and the items that make the whole.
![untitled](https://cloud.githubusercontent.com/assets/10153234/6767498/e750cd9e-d03b-11e4-8f20-5ab5e8cc655c.jpg)

**Comment by Oblivionn** [comment]
Rounding, a point in attack power gives 1.3ap, armor is (1.6?)

**Comment by vasilegherasim** [comment]
Well, i didn't spent any ap on than abillity, the attack power.and the armor i spent 5 points, and says Armor 8 in edit hero mode, so is a round number.

**Comment by uncannymanny** [comment]
Are the numbers in red the floating ones?

Sometimes those are off.

https://github.com/IroncladGames/SinsOfADarkAge/issues/416#issuecomment-41454494

**Comment by vasilegherasim** [comment]
No, those numbers i did added as the correct calculation of ap point.

**Comment by uncannymanny** [comment]
Ah sorry. Yes I misunderstood.

Yes there is rounding involved, which is why the actual math in the tooltip may vary by a point compared to the stats in the bottom left or C.
